keras_cv/layers/preprocessing/grayscale.py

```diff
@@ -50,6 +50,7 @@ class Grayscale(BaseImageAugmentationLayer):
     def __init__(self, output_channels=1, **kwargs):
         super().__init__(**kwargs)
         self.output_channels = output_channels
+        self.auto_vectorize = False
```

Comment:
so grayscale doesn't work on GPU with auto_vectorize=True?

I think we should document this in a comment

Reply:
Done.